credential-management/credentialscontainer-create-basics.https.html

```diff
@@ -140,4 +140,23 @@
     return promise_rejects_exactly(t, "custom reason",
         navigator.credentials.create({ signal: controller.signal }));
 }, "navigator.credentials.create() aborted with custom reason");
+
+promise_test(async function(t) {
+    const reasons = [{}, [], new Error("custom error")];
+
+    for (let reason of reasons) {
+        const result = navigator.credentials.create({ signal: AbortSignal.abort(reason) });
+        await promise_rejects_exactly(t, reason, result);
+  }
+}, "navigator.credentials.create() aborted with differnt objects");
+
+promise_test(function(t) {
+    const error = new Error("custom error");
+    const controller = new AbortController();
+
+    const result = navigator.credentials.create({ signal: controller.signal });
+    controller.abort(error); // aborted after the promise is created
+
+    return promise_rejects_exactly(t, error, result);
+}, "navigator.credentials.create() rejects when aborted a tick later");
```

Comment:
I think usually tick is used in reference to some kind of loop, but here no event loop has had a chance to spin. But it also doesn't really matter.

Reply:
Can you suggest different phrasing?